hspec-discover/src/Test/Hspec/Discover/Run.hs

```diff
@@ -64,6 +64,7 @@ mkSpecModule src conf nodes =
   . showString "{-# OPTIONS_GHC -w -Wall -fno-warn-warnings-deprecations #-}\n"
   . showString ("module " ++ moduleName src conf ++ " where\n")
   . importList nodes
+  . showString "import GHC.Tuple (Unit)\n"
```

Comment:
This will break things for existing users!

`hspec-discover` can only rely on what is provided by the `hspec` package.

Please instead add this import to `Test.Hspec.Discover` and re-export `Unit` from there.